### ISSUE TITLE
Suggestion: Simplify Button Text

### ISSUE THREAD

**Issue by PJSingh5000** [tsujan/Kvantum#581]
Thank you for creating this app.

I have a small suggestion: the instructions should be placed as labels outside the buttons, and the buttons should have a simple action label.

1. Replace "Select a Kavantum theme folder" text in the button, with the word "Select".
2. Replace "Install this theme" text in the button, with "Install" or "Install theme".
3. Finally, add the instructions above the widgets:
"Select a Kavantum theme folder and click the install button to install the theme."
If you find this to be too wordy, simpler instructions may be used: "Select a Kavantum theme folder and click Install"

![image](https://user-images.githubusercontent.com/19394936/102946184-dfc15900-448d-11eb-8673-af77c045eefe.png)

**Comment by tsujan** [comment]
Thanks for your suggestions!

I'm afraid these things depend on people's tastes.

**Comment by tsujan** [comment]
BTW, your screenshot shows that you need a scale factor of 2, instead of a big font.

**Comment by PJSingh5000** [comment]
Yes, I have a HDPI screen so I scale it.

However, I am not talking about font size.

Here is a screen shot of the Kvantium GUI at scale factor 1.

As you can see the button on the left is very large.

Please keep my suggestion in mind for the future. It is a small change that you may find aligns with most modern user interface guidelines.

Thanks for this application, by the way! It is very useful.

![image](https://user-images.githubusercontent.com/19394936/102957688-0ccf3500-44a9-11eb-8bc6-d5fad8ce966f.png)

**Comment by tsujan** [comment]
> However, I am not talking about font size.

I know. That was why I started my second comment with "BTW".

It seems there is a problem in the Qt plugin you use. If the scale factor was 2 in your first screenshot, the icons should have been scaled too. That's the problem you mentioned in https://github.com/tsujan/Kvantum/issues/582 (and I saw it before you reported it).

**Comment by tsujan** [comment]
This is Kvantum Manager with a scale factor of 2:

![scale_factor](https://user-images.githubusercontent.com/981076/102958403-00bb9580-44f2-11eb-93a3-95d4e95adf4e.png)

The same should be seen with all Qt apps: icon and font are *both* scaled.

**Comment by PJSingh5000** [comment]
One more question (and I truly do appreciate your time and patience!)...

Perhaps I could open a bug report with Ubuntu to update or change the QT plugin they ship.

Do you have a suggestion how I would proceed to determine which Qt plugin is being used by my distro?

The packages I have that start with "qt5" are...

- qt5-gtk-platformtheme
- qt5-image-formats-plugins
- qt5-qmltooling-plugins

I presume it is the 1st or the third in the list?

These also depend on a package called "libqt5core5a (>= 5.14.1)".

Which QT plugin do you use?

**Comment by tsujan** [comment]
> Which QT plugin do you use?

I use Qt-based DEs, namely, LXQt and KDE, both of which have advanced Qt plugins with comprehensive GUIs. LXQt's plugin works everywhere (I've tested it under Gnome) but you might need to install extra packages to use it. So, let's forget about it for now.

The most used Qt plugin for GTK DEs like Gnome is called "qt5ct". I'm sure better programs are also made for Gnome but "qt5ct" does the job. I recommend you consult Ubuntu docs/forums on "qt5ct" and how to enable it under Gnome (sorry, I'm not an Ubuntu user); they may also contain info about other available programs.

Long story short, everything was straightforward with GTK2 and Qt4; your didn't need to worry about Qt plugin under Gnome. But that changed with Qt5 and GTK3. Unfortunately, the two toolkits diverged from each other more than before; whether GTK3 was the culprit or Qt5, I don't know.
